src/node_file.cc

```diff
@@ -1649,8 +1649,9 @@ static void RmSync(const FunctionCallbackInfo<Value>& args) {
   // File is a directory and recursive is false
   if (file_status.type() == std::filesystem::file_type::directory &&
       !recursive) {
+    auto file_path_str = ConvertPathToUTF8(file_path);
     return THROW_ERR_FS_EISDIR(
-        isolate, "Path is a directory: %s", file_path.c_str());
+        isolate, "Path is a directory: %s", file_path_str);
```

Comment:
```suggestion
        isolate, "Path is a directory: %s", *path);
```

This is now always assumed to contain UTF8 data.

Reply:
@joyeecheung Yeah. Updated the code and tested locally on Windows and Linux. Let's see what CI says.